src/core/financial.jl

```diff
@@ -388,7 +389,7 @@ function get_EASIUR2005(stack::String; pop_year::Int64=2005, income_year::Int64=
                 setindex!(ret_map, v .* adj, k)
             end
         catch
-            throw(@error("income year is $(income_year) but must be between 1990 to 2024"))
+            throw(@error("EASIUR income year is $(income_year) but must be between 1990 to 2024"))
```

Comment:
also throw an error if pop_year is outside its bounds?

Reply:
Since we're manually setting these (i.e. they're not user inputs) and the pop year goes up to 2050 I think we should be good on that for a while?

Reply:
right!